package.json

```diff
@@ -34,24 +34,24 @@
   },
   "homepage": "https://github.com/dabapps/redux-requests#readme",
   "dependencies": {
-    "@types/js-cookie": "^2.0.28",
-    "@types/node": "^7.0.18",
-    "axios": "^0.18.0",
+    "@types/js-cookie": "^2.2.5",
+    "@types/node": "^10.17.17",
+    "axios": "^0.18.1",
     "flux-standard-action": "^1.2.0",
-    "js-cookie": "^2.1.4",
+    "js-cookie": "^2.2.1",
     "path": "^0.12.7",
-    "redux": "*",
+    "redux": "^4.0.5",
     "redux-thunk": "*"
   },
   "devDependencies": {
-    "@types/jest": "^24.0.11",
-    "jest": "^24.7.1",
-    "prettier": "^1.16.4",
-    "request": "^2.88.0",
-    "ts-jest": "^24.0.2",
-    "tslint": "^5.11.0",
-    "tslint-config-dabapps": "github:dabapps/tslint-config-dabapps#v0.5.3",
-    "typescript": "^3.3.4000"
+    "@types/jest": "^24.9.1",
+    "jest": "^24.9.0",
+    "prettier": "^1.19.1",
+    "request": "^2.88.2",
+    "ts-jest": "^24.3.0",
+    "tslint": "^5.20.1",
+    "tslint-config-dabapps": "^0.6.2",
+    "typescript": "^3.8.3"
   },
   "peerDependencies": {
     "redux-thunk": "*",
```

Comment:
Since you're not changing the peer dependency versioning, is bumping the redux version fixing this issue?

Reply:
(As in the issue described in the trello card)

Reply:
That is right, it fixes the issue.